SignalServiceKit/Utilities/extract_analytics_event_names.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
```

Comment:
Done. On reflection, I think we should continue to use this script as we add more analytics events.